global-institute-of-technology-and-business.csl

```diff
@@ -0,0 +1,550 @@
+    <choose>
+      <if is-numeric="volume">
+        <group>
+          <text term="volume"/>
+          <number variable="volume"/>
+        </group>
+      </if>
+      <else>
+        <text variable="volume" prefix="Volume "/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group>
+          <text term="edition"/>
+          <number variable="edition"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issue">
+    <choose>
+      <if is-numeric="issue">
+        <number variable="issue" prefix="(" suffix=")"/>
+      </if>
+      <else>
+        <text variable="issue" prefix=" "/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="collection">
+    <choose>
+      <if type="report">
+        <group prefix=" " suffix="." delimiter=" ">
+          <text variable="collection-title"/>
+          <text variable="number" prefix=" Report No.: "/>
+        </group>
+      </if>
+      <else>
+        <group prefix=" " suffix="." delimiter=" ">
+          <names variable="collection-editor" suffix=". ">
+            <name delimiter-precedes-last="always" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+            <label prefix=", "/>
+          </names>
+          <text variable="collection-title"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-givenname="true">
+    <sort>
+      <key macro="year"/>
+      <key macro="author-short"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=" ">
+        <text macro="author-short"/>
+        <text macro="year"/>
+      </group>
+      <text variable="locator" prefix=":"/>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="true" et-al-min="11" et-al-use-first="10" entry-spacing="0">
+    <sort>
+      <key macro="author"/>
+      <key macro="year-date"/>
+    </sort>
+    <layout>
+      <group suffix="." delimiter=". ">
+        <text macro="author"/>
+        <text macro="year-date"/>
+        <text macro="title"/>
+      </group>
+      <group suffix=".">
+        <choose>
+          <if type="bill book graphic legal_case legislation motion_picture report song thesis" match="any">
+            <group prefix=" " suffix="." delimiter=" ">
+              <text macro="edition" suffix="."/>
+              <text macro="volume" suffix="."/>
+              <text macro="collection" prefix=" "/>
+              <text macro="editor" suffix="."/>
+            </group>
+            <text prefix=" " macro="publisher"/>
+            <text variable="page" prefix="hlm "/>
+            <text variable="original-title" font-style="italic" prefix=" Terjemahan dari: " suffix="."/>
+            <text variable="note" prefix=" Ed ke-" suffix="."/>
```

Comment:
Can you explain why you are calling the "note" variable? This is the extra field in Zotero and should not be called as people have sometimes large amounts of texts here.
I would suggest to replace this with a specific variable that gets added to the extra field, but I would need to understand what you want to do here to suggest an alternative.